src/commands/profiles/port.js

```diff
@@ -0,0 +1,83 @@
+const { BaseCommand } = require('@twilio/cli-core').baseCommands;
+const { cli } = require('cli-ux');
+
+class ProfilesPort extends BaseCommand {
+  async run() {
+    await super.run();
+
+    const query = await this.inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'confirmation',
+        message:
+          `This command will port profile(s) from keytar` +
+          ` to config file at location ${this.configFile.filePath}. Continue?`,
+        default: false,
+      },
+    ]);
+
+    if (!query.confirmation) {
+      this.logger.warn('Abort!');
+      return;
+    }
+
+    const { projects } = this.userConfig;
+
+    if (this.args.profile) {
+      const sanitizedProfileId = this.args.profile.trim();
+      const project = projects.find((p) => p.id === sanitizedProfileId);
+
+      if (!project) {
+        this.logger.error(
+          `Unable to port profile ${sanitizedProfileId} since it doesn't exist or has already been ported!`,
+        );
+        return;
+      }
+
+      await this.portProfile(project);
+    } else {
```

Comment:
Shouldn't we look at all projects only when ALL option is used. Should this be the default? If yes, then we should update the description to indicate this clearly.